components/camera/transformpipeline/classifier_test.go

```diff
@@ -1,5 +1,3 @@
-//go:build !no_tflite
```

Comment:
cc @hexbabe, now we can probably move the static source to live with its videosourcewrapper counterparts since there's the weird flite test build dependency circus is gone.